### ISSUE TITLE
UnicodeDecodeError for Spanish word `comunicaciones`

### ISSUE THREAD

**Issue by Bender250** [vpodpecan/lemmagen3#2]
I found that Spanish word "comunicaciones" is breaking the lemmatizer. Steps to reproduce:

```py
import lemmagen3

lemmagen = lemmagen3.Lemmatizer('es')
lemmagen.lemmatize('comunicaciones')  # words 'notificaciones', 'conversaciones' crash as well
```

Produces:

```py
     13 def lemmatize(self, word):
     14     '''
     15     Lemmatizes given word.
     16 
   (...)
     25         Lemmatized form of the input word.
     26     '''
---> 27     return self.__lemmatizer.lemmatize(word)

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf3 in position 10: invalid continuation byte
```

Randomly modifying the word did not result in the above given error.

My Python version: Python 3.10.6 (main, Aug 10 2022, 11:40:04) [GCC 11.3.0]

**Comment by vpodpecan** [comment]
Lemmagen returns `0xf3` byte which is ó in latin1 but is invalid in UTF-8. I believe this is because some of the original source files were `latin1` (iso-8859-1) and compiled models returned such strings. Currently, the oldest models are for `es`, `de`, and `it`. The solution would be to recompile models for those languages. The issue, however, is to obtain lexicons because I believe the original ones are licenced.

P.S. Note, however, that `comunicaciónes` is lemmatized into `comunicaciónar` without exceptions.

**Comment by vpodpecan** [comment]
I have rebuilt models for es, de, and fr using `utf-8` sources instead of `iso-8859-1`. The reported bug is now fixed and `comunicaciones` lemmatizes into `comunicación`. The release with the fix (v3.4.0) is already available on [pypi](https://pypi.org/project/lemmagen3/3.4.0/).
Thanks again for reporting the issue.
